src/Components/PostList/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { GoComment } from 'react-icons/go';
+import { AiOutlineStar, AiOutlineUser } from 'react-icons/ai';
+
+import './postlist.css';
+
+function PostList(props) {
+  return (
+    <li className="story-card">
+      <Link to={`/story/${props.post.id}`}>
```

Comment:
.

Reply:
How can that be done? I have /story/:id on constants but I need to pass id here.

Reply:
stroryPath.replace(':id',yourId).